### DISCUSSION TITLE
Ask GitHub copilot

### DISCUSSION THREAD

**Discussion by Biantuadi** [community/community#39266] [GitHub Education] [answered]
Good morning,
Last week I had subscribed for github copilot 10€ per me, and it was last week that I activated github for students and I just learned that copyilot is free for students but yet I was prelifted 10€ today around 8am.
I tried to stop the subscription for me everything was okey; but yet I just received an email with a fee of 80€....
I don’t understand why.
Can GitHub refund me?
thank you.

**Comment by samus-aran** [comment, marked as answer]
Hi @Biantuadi 👋  The best place to go for answers to account-specific questions like this one is opening a ticket on our [Billing and payment Support page](https://support.github.com/features/billing_and_payments). Please open a ticket with them :)
